solver/utils_test.go

```diff
@@ -21,9 +21,15 @@ func Test_PrettyPrintJSONString(t *testing.T) {
 		{
 			name: "basic JSON string",
 			args: args{
-				jstring: "{\"foo\":\"foo\",\"bar\":{\"a\": 1, \"b\": 2}}",
+				jstring: `{"foo":"foo","bar":{"a": 1, "b": 2}}`,
 			},
-			want:    "{\n\t\"bar\": {\n\t\t\"a\": 1,\n\t\t\"b\": 2\n\t},\n\t\"foo\": \"foo\"\n}",
+			want: `{
+	"bar": {
+		"a": 1,
+		"b": 2
+	},
+	"foo": "foo"
+}`,
```

Comment:
If you find this ugly, you could use https://github.com/lithammer/dedent to make this better. We should absolutely do this, not a blocker for merge.